zmap/zgrab2
# zgrab2: Modified custom JSON marshaler

Added a modified copy of the golang `json/encoding` to `lib/json`, whose `Marshal()` takes a `debug` boolean -- and if this boolean is false, fields tagged with `zgrab:"debug"` are omitted from the output.

## How to Test

`make integration-test` -- then check zgrab-output. If `./zgrab-output/mysql/8.0.json` still has `raw_packets`, it's not working.

## Notes & Caveats

See the README.md (https://github.com/zmap/zgrab2/blob/71e386d86dc0e761ec06c111289b19dc7071d66a/lib/json/README.md)

## Issue Tracking

Issue #29
